lib/PlansConfigurator/PlansSearcher.tsx

```diff
@@ -0,0 +1,37 @@
+import { IPlansSearcher } from './interface'
+
+export default function PlansSearcher({ onSearch }: IPlansSearcher) {
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    onSearch?.(e.target.value);
+  };
+
+  return (
+    <div className="mobbex-pc-config-search-input">
+      <span className="mobbex-pc-config-input-title">Seleccionar planes</span>
+      <div className="mobbex-pc-config-input-container">
+        <input
+          className="mobbex-pc-config-input"
+          name="select-plans"
+          id="select-plans"
+          autoComplete="off"
+          placeholder="Buscar"
+          aria-label="Buscar planes"
+          onChange={handleInputChange}
```

Comment:
Elimino este handler ya que se puede realizar en una línea también
```suggestion

  return (
    <div className="mobbex-pc-config-search-input">
      <span className="mobbex-pc-config-input-title">Seleccionar planes</span>
      <div className="mobbex-pc-config-input-container">
        <input
          className="mobbex-pc-config-input"
          name="select-plans"
          id="select-plans"
          autoComplete="off"
          placeholder="Buscar"
          aria-label="Buscar planes"
          onChange={(e) => { onSearch?.(e.target.value) }}
```